scala/lambdas/preingest-tdr-package-builder/src/test/scala/uk/gov/nationalarchives/preingesttdrpackagebuilder/LambdaTest.scala

```diff
@@ -148,7 +171,7 @@ class LambdaTest extends AnyFlatSpec with ScalaCheckDrivenPropertyChecks:
     val contentFolderMetadataObject = metadataObjects.collect { case contentFolderMetadataObject: ContentFolderMetadataObject => contentFolderMetadataObject }.head
     val assetMetadataObject = metadataObjects.collect { case assetMetadataObject: AssetMetadataObject => assetMetadataObject }.head
     val fileMetadataObjects = metadataObjects.collect { case fileMetadataObject: FileMetadataObject => fileMetadataObject }
-    val fileMetadataObject = fileMetadataObjects.filterNot(_.name.endsWith("-metadata.json")).head
+    val fileObjects = fileMetadataObjects.filterNot(_.name.endsWith("-metadata.json"))
```

Comment:
Maybe you could  use `.partition` instead to avoid having to update in two places in the future?